src/preload.py

```diff
@@ -608,13 +608,14 @@ def write_resin_device_pinning(app_data, output):
             ),
         )
 
-
```

Comment:
Re deleted blank lines: if you `npm run lint-python`, which in turn runs `flake8`, I think you will find that the linter will put the blank lines back. :-) I think it is from PEP-8:
> https://www.python.org/dev/peps/pep-0008/#blank-lines
> Blank Lines
> Surround top-level function and class definitions with two blank lines.

Reply:
We were typing at the same time, see my other comment :-)
Is there a valid use case for running `balena preload` multiple times for the same image file?